envs/feedstock-patches/ffmpeg/0001-opence-changes-for-build-with-openssl-for-fips.patch

```diff
@@ -0,0 +1,124 @@
+From 90fb053d9c55eef7ea223e1e5c4a409c530a0218 Mon Sep 17 00:00:00 2001
+From: root <root@dldev7.aus.stglabs.ibm.com>
+Date: Thu, 11 May 2023 05:00:48 -0400
+Subject: [PATCH] opence changes for build with openssl for fips
+
+---
+ recipe/build.sh  | 32 +++++++++++++++++++++++++++-----
+ recipe/meta.yaml | 13 +++----------
+ 2 files changed, 30 insertions(+), 15 deletions(-)
+
+diff --git a/recipe/build.sh b/recipe/build.sh
+index 8528472..887d01b 100755
+--- a/recipe/build.sh
++++ b/recipe/build.sh
+@@ -1,4 +1,5 @@
+ #!/bin/bash
++set -ex
+ 
+ # unset the SUBDIR variable since it changes the behavior of make here
+ unset SUBDIR
+@@ -20,17 +21,22 @@ if [[ ${USE_NONFREE} == yes ]]; then
+   _CONFIG_OPTS+=("--enable-libx264")
+ else
+   _CONFIG_OPTS+=("--disable-nonfree")
+-  _CONFIG_OPTS+=("--enable-gpl")
+-  _CONFIG_OPTS+=("--enable-gnutls")
++  _CONFIG_OPTS+=("--disable-gpl")
```

Comment:
Because `openssl` is not compatible with `gpl`. If `gpl` is on we get the following error:
`openssl is incompatible with the gpl and --enable-nonfree is not specified`. 

so for FIPS build we are using this patch where `openssl` is enabled and `gpl` is disabled.

Reply:
That might disable some codecs that users expect. I guess it's OK to take this route for Open CE 1.9.0, because it's a new release. But we may have to reconsider afterwards if there are GPLed codecs important enough to spend the extra effort on gnutls.

Reply:
Yep, Deepali is currently preparing the "delta" of codecs that we'd mis, and we make the call accordingly...

Reply:
No codec changes. We're good to go!